### DISCUSSION TITLE
Bug Report - Milestones Not showing correct order

### DISCUSSION THREAD

**Discussion by slashtangent** [RetroAchievements/RAWeb#2108] [Ideas]
There is a bug with milestones not showing the correct timeline of events. Using mine as an example: https://retroachievements.org/user/SlashTangent/progress It shows that my first beaten game was Halo 2600 on Jan 6 2021 however, I have games beaten before then. In this instance, I think my first beaten game is Pokemon TCG (GBC) on November 13th, 2020.

**Comment by wescopeland** [comment]
Migrated to https://github.com/RetroAchievements/RAWeb/issues/2109.
